dynatrace-operator/templates/customresource-dynakube.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if and (.Values.apiUrl) (.Values.apiToken) (.Values.paasToken) }}
```

Comment:
According to the [Helm Docu](https://helm.sh/docs/chart_template_guide/function_list/), `and` only takes two arguments, according to the [golang templating docu](https://godoc.org/text/template), it evaluates every argument.

In the case the former is true, it should probably look more like 
```
{{- if and (and (.Values.apiUrl) (.Values.apiToken)) (.Values.paasToken) }}
```

Otherwise, ignore this comment